### PR TITLE
refactor: replace console.logs with client.logger

### PR DESCRIPTION
**Pull request by itsUrcute**

### CODE REVIEW

File: src/classes/Logger.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ export class Logger {
 		} else {
 			timeString = "";
 		}
-		console.log(colorFn(`[ ${modifier}${timeString} ] ${input}`));
+		console.log(colorFn(`[ ${modifier}${timeString} ] `) + input);
```
Comment (cheeze2000) [inline src/classes/Logger.ts:29]: why

File: src/utils/loadFiles.ts [hunk 2]
```diff
@@ -1,21 +1,26 @@
...
+	collection: Collection<string, T>,
+	path: string,
+	client: Client
+): Promise<void> {
 	const files = await readdir(path, { withFileTypes: true });
 	for (const element of files) {
 		if (element.isFile() && element.name.endsWith(".js")) {
 			const imp: { default: T } = await import(join(path, element.name));
 			collection.set(imp.default.data.name, imp.default);
+			client.logger.debug(`${imp.default.data.name} loaded successfully.`);
```
Comment (cheeze2000) [inline src/utils/loadFiles.ts:21]: is it possible to separate these strings in a constants file

File: src/classes/Logger.ts (hunk 1 above)
Comment (itsUrcute) [reply src/classes/Logger.ts:29]: ask @TogarashiPepper  ¯\_(ツ)_/¯

Comment (TogarashiPepper) [reply src/classes/Logger.ts:29]: because that way only the text within `[]` is colored I suppose

Comment (cheeze2000) [reply src/classes/Logger.ts:29]: oke, can we change to `), input` instead of `) + input`

Comment (itsUrcute) [reply src/classes/Logger.ts:29]: sure

**Pull request merged**